src/pages/ListIncidents.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <div>
+        <h1>Incident Reports</h1>
```

Comment:
Currently, our transaltors could not translate this string.
Please use `{{ $t("Incident Reports") }}` instead and add this key to the `en.json`.

There are a few other cases of this as well